ui/app/rif/pages/domainsDetailPage/domainsDetailPage.js

```diff
@@ -40,9 +32,7 @@ function mapStateToProps (state) {
 }
 
 const mapDispatchToProps = dispatch => {
-	return {
-		goBack: () => dispatch(rifActions.navigateTo(pageNames.rns.domains)),
-	}
+	return {}
```

Comment:
If we are not using any action binded with redux, we can just omit this. `connect ` accepts passing only `mapStateToProps`

Reply:
I think that it's going to be changed by @rorolopetegui so i guess it's not going to be empty

Reply:
Yep, in next PRs we're going to use the Connect with botch parameters